pkgs/applications/editors/android-studio/common.nix

```diff
@@ -175,9 +175,7 @@ let
 
           # For GTKLookAndFeel
           gtk2
```

Comment:
We should really switch to GTK 3 but I will leave it to someone else.

Reply:
I have gtk2 marked broken in my local config already 😅